package.json

```diff
@@ -70,7 +70,7 @@
   },
   "dependencies": {
     "bindings": "^1.5.0",
-    "bson": "^4.0.3",
+    "bson": "github:steffenagger/js-bson#uuid-realm",
```

Comment:
Is this the final value or would this update before this PR is merged?

Reply:
Ideally this links to a released bson-version, but I'm not sure that will happen in time.